.gitignore

```diff
@@ -54,3 +57,6 @@ build/
 /application.yaml
 /application.properties
 .Rproj.user
+
+# Rstudio
+molgenis-service-armadillo.Rproj
```

Comment:
I don't think R studio project belong in the project root opened as we have at least 2 different R projects in scripts/